### PR TITLE
MB-1804 Containers

### PR DESCRIPTION
**Pull request by Shkeating**
## Description

In this branch, we create classes for the different kinds of containers in our design system and provide them with styling. Includes the following:
- Container Default
- Container Gray
- Container Popout
- Container Accent

## Reviewer Notes
The spec didn't include margin or padding values. I've set them both to the USWDS token of `1` (24px) for the moment. Let me know if we want to change that.

## Setup

The usual

```sh
yarn storybook
```

or

```sh
make storybook_docker
```

if that branch has been pushed to master

## Code Review Verification Steps
- [ ] User facing changes have been reviewed by design.
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## References

* [JIRA story](https://dp3.atlassian.net/browse/MB-1804?atlOrigin=eyJpIjoiNzZiOWQ0OTRmM2E1NDIzZThmYjY0ZGE0NTUwNTllZTMiLCJwIjoiaiJ9) 
* [Abstract Sticker Sheet](https://share.goabstract.com/480df22a-d2d9-4ffd-a673-21fcc337f342?collectionLayerId=fae86d86-9306-4212-a813-ecee367bf643&mode=build) 

## Screenshots

image

### CODE REVIEW

File: src/shared/styles/_mixins.scss [hunk 1]
```diff
@@ -1,24 +1,29 @@
 //Mixins
+@import 'uswds';
```
Comment (gidjin) [inline src/shared/styles/_mixins.scss]: Why do you need to import uswds here?

Comment (Shkeating) [reply src/shared/styles/_mixins.scss]: u right tho

Comment (Shkeating) [reply src/shared/styles/_mixins.scss]: i took it out

**Comment by gidjin** [comment]
The job that will check for this isn't required quite yet, but you will need to update the loki tests for these changes. Let me know if you need help. The docs in https://github.com/transcom/mymove/blob/master/docs/how-to/run-loki-tests-against-storybook.md should help.

**Comment by samadden** [comment]
1) **Container default** is not currently displaying a border; instead it should have a 1px border of $base-lighter

2) **Container popout** is displaying a 2px border ; instead it should be a 1px border of $base-light

**Pull request merged**